src/main/components/uml-element/assessable/assessable.tsx

```diff
@@ -170,11 +170,12 @@ export const assessable = (
         // TODO: following messages should be received from Artemis
         const removeMessage = 'Do you want to remove the link to the assessment instruction?';
         const tooltipMessage = 'Assessment Instruction: ' + instruction.instructionDescription;
+        const feedbackHint =
+          'This feedback is associated with the assessment instruction. You can provide additional feedback for this submission element. Student will see combined feedback during the review.';
```

Comment:
Actually, Artemis send these messages when the modeling editor initialized for already existing feedback. I only added these messages in Apollon, when the instruction drag&drop into feedback element. This drag&drop event functionality needs to be refactored between Artemis and Apollon. Imo, solution for this can be covered another PR and I am open any suggestions.

Reply:
I agree. This can be done in another PR.